ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -80,82 +80,86 @@
             <B.Tr>
               <B.Td>
                 <span class='hds-font-weight-semibold'>
-                  {{#if (can 'read model' B.data.target)}}
+                  {{#if (can 'read model' B.data)}}
                     <LinkTo
-                      data-test-visit-target={{B.data.target.id}}
+                      data-test-visit-target={{B.data.id}}
                       @route='scopes.scope.projects.targets.target'
-                      @model={{B.data.target.id}}
+                      @model={{B.data.id}}
                       @query={{hash isConnecting=false}}
                     >
-                      {{B.data.target.displayName}}
+                      {{B.data.displayName}}
                     </LinkTo>
                   {{else}}
-                    {{B.data.target.displayName}}
+                    {{B.data.displayName}}
                   {{/if}}
-                  {{#if B.data.target.description}}
-                    <p>{{B.data.target.description}}</p>
+                  {{#if B.data.description}}
+                    <p>{{B.data.description}}</p>
                   {{/if}}
                 </span>
               </B.Td>
               <B.Td>
                 <Copyable
-                  @text={{B.data.target.id}}
+                  @text={{B.data.id}}
                   @buttonText={{t 'actions.copy-to-clipboard'}}
                   @acknowledgeText={{t 'states.copied'}}
                 >
-                  <code>{{B.data.target.id}}</code>
+                  <Hds::Text::Code>{{B.data.id}}</Hds::Text::Code>
                 </Copyable>
               </B.Td>
               <B.Td>
-                {{#if B.data.sessions.length}}
-                  <Hds::Button
-                    @text={{t 'actions.yes'}}
-                    @color='tertiary'
-                    @icon='info'
-                    {{on 'click' this.toggleSessionsFlyout}}
-                  />
-                {{/if}}
+                {{#let
+                  (call (fn this.getAvailableSessions B.data))
```

Comment:
Is this needed? You only call this to sort the sessions but it only gets used on line 114 it looks like so do you really need it sorted?

Couldn't you just call `{{#if B.data.isActive}}` on line 114 instead?

Reply:
ahh right, that is much simpler!